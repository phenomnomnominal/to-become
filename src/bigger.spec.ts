import { bigger } from './index';

describe(`toBecome(bigger)`, () => {
    it(`it should work for values that grow`, () => {
        expect(Date.now()).toBecome(bigger);
    });

    it(`it should work for values that stay the same size`, () => {
        expect(1000).toBecome(bigger);
    });

    it('should not break existing snapshots', () => {
        expect(1000).toMatchSnapshot();
    });
});
