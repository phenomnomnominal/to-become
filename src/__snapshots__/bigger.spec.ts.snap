// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toBecome(bigger) it should work for values that grow 1`] = `1567468712582`;

exports[`toBecome(bigger) it should work for values that stay the same size 1`] = `1000`;

exports[`toBecome(bigger) should not break existing snapshots 1`] = `1000`;
